src/libponyrt/sched/start.c

```diff
@@ -148,6 +152,17 @@ PONY_API int pony_init(int argc, char** argv)
     exit(0);
   }
 
+  if (opt.noscale)
+  {
+    opt.min_threads = opt.threads;
+  }
+
+  if (opt.min_threads > opt.threads) 
+  {
+    printf("Can't have --ponyminthreads > --ponythreads (%d > %d)\n", opt.min_threads, opt.threads);
```

Comment:
Looks like other errors aren't on `stderr`, but should they be?
Also is `-1` as a return fine?